tfhe/src/integer/gpu/server_key/radix/scalar_comparison.rs

```diff
@@ -120,6 +120,144 @@ impl CudaServerKey {
         result
     }
 
+    /// # Safety
+    ///
+    /// - `stream` __must__ be synchronized to guarantee computation has finished, and inputs must
+    ///   not be dropped until stream is synchronised
```

Comment:
@tmontaigu we don't add doc tests for every entry point? We're missing lots of doc tests I think, not sure if it's bad?

Reply:
No, my thinking on this is that, 'default' operations (the ones without prefix unchecked/smarts should generally  have  a doctest as its what we direct user towards.

unchecked/smart do not really need them, they are a bit more advanced, and so the user should already know how to call them.